app/src/main/assets/task/Tapping.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
Lets move these into motor-control-module too. I think asset merging should merge it into app, that would be the ideal way for future apps to add a module